controllers/cloud.redhat.com/run.go

```diff
@@ -29,6 +30,9 @@ var (
 	setupLog = ctrl.Log.WithName("setup")
 )
 
+var mu sync.RWMutex
```

Comment:
do you think it makes more sense to define these vars in `clowdenvironment_controller.go`?